src/www/diag_system_pftop.php

```diff
@@ -150,7 +150,7 @@ function getpftopactivity() {
                       <option value='200' selected="selected">200</option>
                       <option value='500'>500</option>
                       <option value='1000'>1000</option>
-                      <option value='9999999999'>all</option>
+                      <option value='9999999999'><?= gettext('all') ?></option>
```

Comment:
Unrelated side note :D

    $infinity =  9999999999;
    echo "To $infinity and beyond.";